src/resources/birthdayTimer.ts

```diff
@@ -0,0 +1,61 @@
+import { bdayDates, bdayTimes, bdayChannels } from '../slashcommands/Birthday';
+import cron from 'node-cron';
+import { Bot } from '../Bot';
+import { MessageEmbed, TextChannel, User } from 'discord.js';
+import Enmap from 'enmap';
+
+export let bdayCrons = new Enmap('bdayCrons');
```

Comment:
Instead of making this a named enmap and deleting the entire enmap when starting up, make it a non-named enmap so you don't have to delete everything. Only named enmaps save data between executions. Since you're getting rid of the data right away, there's no need to save it.